null_safety_examples/null_safety_codelab/bin/more_nullable_types.dart

```diff
@@ -0,0 +1,9 @@
+void main() {
+  List<String> aListofStrings = ['one', 'two', 'three'];
+  List<String?> aNullableListOfStrings = [];
+  List<String?> aListofNullableStrings = ['one', null, 'three'];
```

Comment:
Seems like `of` should be `Of` to be consistent with the middle declaration and general naming standards.